src/OpenTelemetry.Exporter.Jaeger/JaegerExporter.cs

```diff
@@ -15,47 +15,51 @@
 // </copyright>
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Threading;
 using System.Threading.Tasks;
 using OpenTelemetry.Exporter.Jaeger.Implementation;
 using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
 using OpenTelemetry.Trace.Export;
 
 namespace OpenTelemetry.Exporter.Jaeger
 {
-    public class JaegerTraceExporter : SpanExporter, IDisposable
+    public class JaegerExporter : ActivityExporter, IDisposable
     {
         private readonly SemaphoreSlim exportLock = new SemaphoreSlim(1);
         private bool libraryResourceApplied = false;
         private bool disposedValue = false; // To detect redundant dispose calls
 
-        public JaegerTraceExporter(JaegerExporterOptions options)
+        public JaegerExporter(JaegerExporterOptions options)
         {
             this.JaegerAgentUdpBatcher = new JaegerUdpBatcher(options);
         }
 
         internal IJaegerUdpBatcher JaegerAgentUdpBatcher { get; }
 
-        public override async Task<ExportResult> ExportAsync(IEnumerable<SpanData> otelSpanList, CancellationToken cancellationToken)
+        public override async Task<ExportResult> ExportAsync(IEnumerable<Activity> activityBatch, CancellationToken cancellationToken)
         {
             await this.exportLock.WaitAsync().ConfigureAwait(false);
             try
             {
-                if (!this.libraryResourceApplied && otelSpanList.Count() > 0)
+                if (!this.libraryResourceApplied && activityBatch.Count() > 0)
                 {
-                    var libraryResource = otelSpanList.First().LibraryResource;
+                    var libraryResource = activityBatch.First().GetResource();
 
                     this.ApplyLibraryResource(libraryResource ?? Resource.Empty);
 
                     this.libraryResourceApplied = true;
                 }
 
-                foreach (var s in otelSpanList)
+                foreach (var activity in activityBatch)
                 {
+                    // TODO: group by activity source
```

Comment:
@eddynaka q.: why?

Reply:
@pjanotti , this comment already existed in JaegerActivityExporter. Don't know the reason.

Reply:
🤦 git blame says that it was added by me... not sure why I added that... could you please remove it?

Reply:
sure! will update soon